### DISCUSSION TITLE
Computed Columns color & background color in CSV download

### DISCUSSION THREAD

**Discussion by Sandy381** [fbaligand/kibana-enhanced-table#289] [General]
Hi  ,

I'm creating Enhanced table with computed columns . I'm creating Background colour with computed condition that if value exists in another column, then color it is red or green.

I'm able to execute this condition in dashboard & able to see colors in dashboard.

But if I download CSV report, I'm unable to see Background color or font color that is in Enhanced table.

I'm using OSD 1.2.0.

Please find below screenshots.

![Screenshot 2022-09-15 at 4 20 13 PM](https://user-images.githubusercontent.com/74658016/190388339-360fd85a-732b-4ec8-a7a8-ed826024b864.png)
![Screenshot 2022-09-15 at 4 19 18 PM](https://user-images.githubusercontent.com/74658016/190388358-db2d15e1-eace-4aca-8853-68b523ed12c6.png)

Anyway to also download background colors and font colors in CSV file.
Thanks in Advance.

**Comment by fbaligand** [comment]
Well, CSV export is not Excel export. 
CSV export is a simple text format, with no rendering metadata for each cell.
So, sorry, but it’s not possible to export background color in CSV format.
